layers/msgpack/Dockerfile

```
@@ -0,0 +1,11 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION AS ext
+
+RUN pecl install --force msgpack
```

I'm pretty sure that you don't need `--force` option.
```suggestion
RUN pecl install msgpack
```